backend/backend/production_settings.py

```diff
@@ -12,29 +12,23 @@
 
 import os
 from pathlib import Path
+from .settings import *
+import sys
+import mimetypes
 
-# Build paths inside the project like this: BASE_DIR / 'subdir'.
-BASE_DIR = Path(__file__).resolve().parent.parent
+mimetypes.add_type("text/css", ".css", True)
 
+TESTING = sys.argv[1:2] == ['test']
```

Comment:
I'd suggest using an argument parser here, or better using an environment variable, In any case, why do you need "TESTING" inside the production settings?

Reply:
First, this is specifically here to flag whenever the command `python manage.py test` is being run, so the check is accurate. We certainly _could_ use an environment variable, but that would add boilerplate to every time we run the test command.

As for why it's necessary in the production settings, those settings are used when making the production image. On the CI, we actually test directly against that image for the most accurate results.